apm-agent-plugins/apm-httpclient11-plugin/src/main/java/co/elastic/apm/agent/httpclient/HttpClientHelper.java

```diff
@@ -0,0 +1,22 @@
+package co.elastic.apm.agent.httpclient;
+
+import co.elastic.apm.agent.bci.VisibleForAdvice;
+
+import javax.annotation.Nullable;
+
+@VisibleForAdvice
+public class HttpClientHelper {
```

Comment:
Rename so the name doesn't clash with `co.elastic.apm.agent.http.client.HttpClientHelper`

Reply:
renamed